static/docs/commands-reference/install.md

```diff
@@ -35,12 +35,24 @@ is a reminder to run `dvc commit`.
 The installed Git hook automates reminding the user to run either `dvc repro` or
 `dvc commit`.
 
+**Push** Sharing data across environments and making changes to remote storage
+is done by `dvc push`. This command searches for all DVC-files from current
+workspace and stages them for consultation. Using checksums it finds for
```

Comment:
it all sounds too complicated, especially ` stages them for consultation`.

It takes all DVC-files from the current workspace, indeed. It checks if some of the data files or directories that are referenced in those DVC-files are missing on the remote storage. And it uploads them.

Let's simplify it all. `Takes DVC-files from the current workspace and saves (if they are not saved yet) files referenced in those to the remote storage.`. Something like this, please :)